app-games/src/main/java/cm/aptoide/pt/app_games/home/MainView.kt

```diff
@@ -149,5 +108,15 @@ private fun NavigationGraph(
       navigateBack = navController::popBackStack,
     )
 
+    allCategoriesScreen(
+      navigateBack = navController::navigateUp,
+      navigate = navController::navigate
+    )
+
+    categoryDetailScreen(
```

Comment:
This screen does not exist at the time of this commit, since it was only added in the next commit. If you interactively rebase and stop at this commit, the project won't compile. It should be added in the following commit